surfsense_web/app/dashboard/[search_space_id]/podcasts/podcasts-client.tsx

```
@@ -0,0 +1,787 @@
+} from "@/components/ui/dialog";
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { toast } from "sonner";
+
+interface Podcast {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Podcast'. Consider to delete it or rename it.

'Podcast' is defined here:


(lint/suspicious/noRedeclare)